options/glibc/include/sys/user.h

```diff
@@ -42,4 +42,8 @@ struct user {
 }
 #endif
 
+#define PAGE_SHIFT 12
```

Comment:
What is this and is it platform dependent?

Reply:
It's apparently a define for gdb, and it is not platform-dependent (in glibc + musl). It does not reflect true page size, as you should use the function for that.